### ISSUE TITLE
Is it possible to leave comments intact?

### ISSUE THREAD

**Issue by TJM** [geofffranks/spruce#353]
We have been trying to use this tool to apply a simple change to a helmrelease file, but there are two things happening that are causing us pain..

* The file is completely re-ordered  - this obviously doesn't change the end results, but it makes for an ugly commit, when I just wanted to change one value
* All comments are stripped

I am assuming it is just reading in the data and merging them programmatically, then writing back out a new yaml file, with no knowledge of the original file, rather than modifying the original file with the changes from the others?

~tommy

**Comment by geofffranks** [comment]
You're correct in why that's happening. Unfortunately it is unlikely to change.

Something like [ytt](https://github.com/vmware-tanzu/carvel-ytt) might be more what you're looking for, but i don't know how it deals with comments?
